docs/modules/publisher.md

```diff
@@ -344,15 +344,49 @@ Returns the nodes configured for one Endpoint (Dataset)
 
 #### **GetDiagnosticInfo_V1**
 
-*TODO: add description*
+Returns a list of actual metrics for every DataFlow Processing Engine being processed by the publisher. 
```

Comment:
Users might not be familiar with our internal definitions such as `DataFlowProcessingEngine`. Can you please modify this so that a person that is not familiar with our codebase understands what to expect ?

Reply:
changed